code/modules/mob/dead/observer/observer.dm

```diff
@@ -304,15 +304,6 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 /mob/dead/observer/is_active()
 	return 0
 
-/mob/dead/observer/Stat()
```

Comment:
would be nice to come up with something to display blob progress to observers

Reply:
I already added it to analyzers for blobs, I'll add it to examining as a ghost as well

Reply:
yea, that works